resources/windows/wix/service.component.wxs.erb

```diff
@@ -0,0 +1,58 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+  <%- get_services.each do |service| -%>
+    <%- service.options.each do |component| -%>
+    <DirectoryRef Id="<%= component[:directory_ref] %>">
+      <Component Id='<%= component[:id] %>' Guid="<%= component[:guid] %>" <%= "Win64=\"$(var.Win64)\"" unless component[:win64].nil? %>>
+        <%- if component[:file][:id] == "NSSM" -%>
+        <CreateFolder />
```

Comment:
Also, I have no idea why this is getting called for the pxp-agent service, but not for the others. The conditional is rather arbitrary, but it'd be nice to have something more concrete to define if we add this line or not